POLICY_CONFIGURATION.md

```diff
@@ -6,6 +6,9 @@ The following issues have been added to the policies exclusion list
 
 | CVE Report    |Type      | Component | Reason       | Date |
 | ------------- | -------  |----------| ------------- | -----------------  |
-|[CVE-2021-33560](https://nvd.nist.gov/vuln/detail/CVE-2021-33560)| Debian | [libgcrypt20](https://security-tracker.debian.org/tracker/CVE-2021-33560) | Debian marked "wont-fix". Do not believe any projects use affected functionality. | Updated 04/04/2022 |
-|[CVE-2011-3389](https://nvd.nist.gov/vuln/detail/CVE-2011-3389)| Debian | [gnutls28](https://security-tracker.debian.org/tracker/CVE-2011-3389) | "No mitigation for gnutls, it is recommended to use TLS 1.1 or 1.2 which is supported since 2.0.0". 1.0 should not be in use by servers our software connects to. | Updated 04/04/2022 |
-|[CVE-2020-16156](https://nvd.nist.gov/vuln/detail/CVE-2020-16156)| Debian | [perl](https://security-tracker.debian.org/tracker/CVE-2020-16156) | Debian marked "wont-fix". Not used at runtime in any services. | Updated 04/04/2022 |
+|[CVE-2021-33560](https://nvd.nist.gov/vuln/detail/CVE-2021-33560)| Debian | [libgcrypt20](https://security-tracker.debian.org/tracker/CVE-2021-33560) | Debian marked "wont-fix". Do not believe any projects use affected functionality. | Updated 20/05/2022 |
+|[CVE-2011-3389](https://nvd.nist.gov/vuln/detail/CVE-2011-3389)| Debian | [gnutls28](https://security-tracker.debian.org/tracker/CVE-2011-3389) | "No mitigation for gnutls, it is recommended to use TLS 1.1 or 1.2 which is supported since 2.0.0". 1.0 should not be in use by servers our software connects to. | Updated 20/05/2022 |
+|[CVE-2020-16156](https://nvd.nist.gov/vuln/detail/CVE-2020-16156)| Debian | [perl](https://security-tracker.debian.org/tracker/CVE-2020-16156) | Debian marked "wont-fix". Not used at runtime in any services. | Updated 20/05/2022 |
+|[CVE-2019-8457](https://nvd.nist.gov/vuln/detail/CVE-2019-8457)| Debian [sqlite3](https://security-tracker.debian.org/tracker/CVE-2019-8457) | Debian marked "wont-fix". "Affected function is not used in Debian and meant for debugging purposes, backporting the fix would be very complex." | Updated 20/05/2022 |
```

Comment:
The new lines are missing a `|` after Debian so the table columns are lining up correctly.

Reply:
Whoops, double checked in markdown preview this time! Fixed by 7c5b338